### ISSUE TITLE
Female Focused Symptoms/ Conditions Updating 5

### ISSUE THREAD

**Issue by Lucy-WhiteSwan** [obophenotype/human-phenotype-ontology#7515]
**Preferred term label:**
Bleeding During Sex

**Synonyms**

**Definition (free text, please give PubMed ID)**
Bleeding during sex, PubMed ID: unknown

**Parent term (use hpo.jax.org/app)**
Abnormal bleeding HP:0001892

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
n/a

**Your nano-attribution (ORCID)**
n/a

**Comment by pnrobinson** [comment]
added as Vaginal bleeding during sex
